app/views/admin/users/index.html.erb

```diff
@@ -100,10 +100,14 @@
 
             <span class="state">
               <i>State:</i> <%= user.state %>
-            </span>
+            </span> |
+
+            <span class="uuid">
+              <i>UUID:</i> <%= user.uuid || '(no uuid)' %>
+            </span> |
 
             <% if user.self_reported_school.present? %>
-              | <span class="school">
```

Comment:
The `|` still needs to be here and not after UUID in case there's no `self_reported_school`

Reply:
That makes sense. Putting it back there. :)